AsyncAwait.Tasks/AsyncAwait.Task2.CodeReviewChallenge/Controllers/HomeController.cs

```diff
@@ -25,15 +26,15 @@ public ActionResult Index()
         return View();
     }
 
-    public ActionResult Privacy()
+    public async Task<IActionResult> Privacy()
     {
-        ViewBag.Message = _privacyDataService.GetPrivacyDataAsync().Result;
+        ViewBag.Message = await _privacyDataService.GetPrivacyDataAsync().ConfigureAwait(false);
         return View();
     }
 
-    public async Task<IActionResult> Help()
+    public ActionResult Help()
     {
-        ViewBag.RequestInfo = await _assistant.RequestAssistanceAsync("guest").ConfigureAwait(false);
+        ViewBag.RequestInfo = _assistant.RequestAssistanceAsync("guest").Result;

```

Comment:
please read the following https://learn.microsoft.com/en-us/dotnet/csharp/asynchronous-programming/async-scenarios#important-info-and-advice

use async\await approach

Reply:
Hi Valery I have update commit last commit